src/Client.php

```diff
@@ -93,4 +93,21 @@ protected function getLastError(): array
 
         return compact('code', 'message');
     }
+
+    /**
+     * Queries the connected printer and returns a PrinterStatus object
+     *
+     * @return null|PrinterStatus
+     */
+    public function getPrinterStatus(): ?PrinterStatus
+    {
+        $zpl = "~HS";
+        $this->send($zpl);
+
+        if (!socket_recv($this->socket, $response, 96, 0) > 0) {
```

Comment:
`socket_recv` will either return a false (if there's an error) or an integer showing the number of bytes received. I don't think this if statement will ever evaluate to true.

```suggestion
        if (!socket_recv($this->socket, $response, 96, 0)) {
```

^^ this would evaluate to true if a `FALSE` or a `0` is received, which is when you want to return a `NULL`

Reply:
Applied. Thanks!